src/Compilers/Core/Portable/Desktop/DesktopStrongNameProvider.cs

```diff
@@ -17,29 +17,91 @@ namespace Microsoft.CodeAnalysis
     /// </summary>
     public class DesktopStrongNameProvider : StrongNameProvider
     {
-        private sealed class TempFileStream : FileStream
+        private sealed class TempFileStream : Stream
```

Comment:
Is this thing only ever used synchronously? Because this will be terribly inefficient for async.